spec/models/loader/types.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+describe Loader::Types do
+  context "Check user creation verification" do
```

Comment:
As we're unit testing, it can be helpful to define the method before the context:
```rb
describe Loader::Types do
  describe '.check_user_creation_allowed' do
    context 'when user creation is disabled' do
       ...
```
Also, context should clearly describe the state we're testing

Reply:
done